recipes/pyparsing/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version="2.1.1" %}
+
+package:
+  name: pyparsing
+  version: {{ version }}
+
+source:
+  fn: pyparsing-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/p/pyparsing/pyparsing-{{ version }}.tar.gz
+  md5: 5ce9096d94f553e2bc1fd366fba65558
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - pyparsing
+
+about:
+  home: http://pyparsing.wikispaces.com/
+  license: MIT
+  license_file: LICENSE
+  summary: Create and execute simple grammars
+  description: |
+    The pyparsing module is an alternative approach to creating and executing
+    simple grammars, vs. the traditional lex/yacc approach, or the use of
+    regular expressions. The pyparsing module provides a library of classes
+    that client code uses to construct the grammar directly in Python code.
+  doc_url: http://pyparsing.wikispaces.com/Documentation
```

Comment:
They list a different link on PyPI, but I like this one better if we are to have one.